### DISCUSSION TITLE
Migrate from 800 Controller to 700 Controller?

### DISCUSSION THREAD

**Discussion by Botched1** [zwave-js/zwave-js#6877] [Q&A] [answered]
Should it be technically possible to backup NVM FROM an 800 controller and retore it TO a 700 controller?

I tried, and it didn't work. But I was curious if it should have worked or not.

**Comment by AlCalzone** [comment]
Yes, but not with PC Controller, only Z-Wave JS. What exactly did not work? Which SDK versions did you migrate between?

**Comment by Botched1** [comment]
What I tried:

* Zwave JS UI backup NVM on Zooz ZSTR39 device running 7.21.3. Backup said it completed successfully, and the downloaded BIN file is ~40KB in size.
* Shutdown Zwave JS UI addon in Home Assistant.
* Removed Zooz ZST39 usb device
* Inserted SiLabs UZB7 device running 718.8.
* Restarted Zwave JS UI addon in Home Assistant
* Changed USB serial settings in Zwave JS UI to the UZB7 device-id
* Soft reset on controller
* Verified controller appeared, as expected
* Attempted NVM Restore on controller using the previous backup

**Reply by AlCalzone** [reply, marked as answer]
That sounds like it should work. How did it fail?

**Reply by Botched1** [reply]
Good grief....

I just tried it again on a 2nd UZB7 I have, and it completed successfully / no errors. Not sure what happened the 1st time on the other stick...

**Reply by Botched1** [reply]
Last questions...

1. I don't know what the default power settings are for a UZB7 in the US... It is set at 6 and 0 right now, but that is what it was on the ZST39...

Edit disregard below. Fixed itself after a restart.

<s>2. Should I be concerned that the zwave js ui still says it is a ZST39? It definitely is not - I only have 1x of them and it is in my hand right now. :)</s>

**Reply by Botched1** [reply]
> That sounds like it should work.

So it did work the 2nd try... MOSTLY.

None of the ZWave LR devices came over. All of the mesh ones did, but none of the LR ones. I did verify the region was set to USA Long Range. Also tried changing to US, save, and back to US Long Range.

I just went around and excluded all of them and re-paired them.

EDIT: This may be something that was fixed in https://github.com/zwave-js/node-zwave-js/pull/6846 . As I'm using zwave js ui, I'm still at 12.5.6.

**Reply by AlCalzone** [reply]
Ahh yes, being on an up to date version is always a good idea when dealing with NVM backup/restore for new SDK versions.
Things change and Z-Wave JS has to adapt sometimes.

**Reply by AlCalzone** [reply]
> I don't know what the default power settings are for a UZB7 in the US

AFAIK they are set to 0/0. Although technically -1 (first value) is supposed to be the maximum allowed powerlevel in US.
